cheese/data/__init__.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass

```

Comment:
You can use ```git mv <oldpath> <newpath>``` to preserve git history when renaming directories and files (just for future reference). In this case the command would be ```git mv backend/data/__init__.py cheese/data/__init__.py```.